frontend/app_flowy/packages/appflowy_editor/lib/src/service/shortcut_event/built_in_shortcut_events.dart

```diff
@@ -51,14 +51,25 @@ List<ShortcutEvent> builtInShortcutEvents = [
   ),
   ShortcutEvent(
```

Comment:
I think there is no need to add more built-in commands for word selection now. I prefer to provide a shortcut commands panel if the users want to modify them.

```dart
   key: 'Cursor down select',
    command: 'shift+alt+arrow left,meta+alt+arrow left',
    windowsCommand: 'shift+alt+arrow left,ctrl+alt+arrow left',
    linuxCommand: 'shift+alt+arrow left,ctrl+alt+arrow left',
    handler: cursorLeftWordSelect,
  ),
  ShortcutEvent(
    key: 'Cursor down select',
    command: 'shift+alt+arrow right,ctrl+alt+arrow right',
    windowsCommand: 'shift+alt+arrow right,ctrl+alt+arrow right',
    linuxCommand: 'shift+alt+arrow right,ctrl+alt+arrow right',
    handler: cursorRightWordSelect,
  ),
```

Reply:
Sure